VoodooSMBus/VoodooSMBusControllerDriver.hpp

```diff
@@ -54,6 +54,16 @@ class VoodooSMBusControllerDriver : public IOService {
     IOWorkLoop* getWorkLoop();
     void handleInterrupt(OSObject* owner, IOInterruptEventSource* src, int intCount);
 
+    /**
+     * readByteData - SMBus "read byte" protocol
+     * @client: Handle to slave device
+     * @command: Byte interpreted by slave
+     *
+     * This executes the SMBus "read byte" protocol, returning negative errno
+     * else zero on success.
```

Comment:
I think this comment is wrong, it returns the actual data byte

Reply:
https://github.com/torvalds/linux/blob/master/drivers/i2c/i2c-core-smbus.c#L120 It is read byte. ReadByte is "Receive Byte" if you scroll up in the link.

Reply:
Wait, I read it read, shoot. I see the error now. It's in Return, not the protocol